grants/milestone_deliveries/season_2/package_manager/data/videos/README.md

```diff
@@ -0,0 +1,10 @@
+# Videos
+
+These research videos compliment the written content. They are saved to both YouTube and Filecoin.
+
+## YouTube
+- [Video 1](https://youtu.be/RmfZo2QdV0A)
+- [Video 2](https://youtu.be/8aF2febVy7A)
```

Comment:
Around 19:05:
* https://youtube.com/watch?v=8aF2febVy7A&feature=shares&t=1149
* You noted that the the git+https method has a "fragility" in cases where a seed node is down. 
* Question: this sounds like the same problem set as site reliability. As a thought experiment, would you agree with this: assuming there was an infinite number of seed nodes, there would - practically speaking - be no fragility? In other words, how fragile this method is is a function of the number of seed nodes a binary is hosted on?

Reply:
I think the answer depends on the behavior of Seed nodes:

- If the nodes share information to resolve a request for a git repository, then more nodes would lead to less fragility.
- If it is expected to go to a specific Seed node to get a repository, then having more nodes would not decrease fragility.

Hopefully someone more familiar with the networking of Seed nodes can point out which answer reflects reality.